es/es-dashboards/pom.xml

```diff
@@ -24,12 +24,74 @@
   <modelVersion>4.0.0</modelVersion>
   <artifactId>gn-es-dashboards</artifactId>
   <name>GeoNetwork dashboard app based on Kibana</name>
-
   <parent>
     <artifactId>gn-es</artifactId>
     <groupId>org.geonetwork-opensource</groupId>
     <version>4.4.4-SNAPSHOT</version>
   </parent>
+
+  <build>
+    <plugins>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-enforcer-plugin</artifactId>
```

Comment:
How does this work?

I've tried: `mvn enforcer:enforce` and get the following error:

```
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-enforcer-plugin:3.4.1:enforce (default-cli) on project gn-es-dashboards: No rules are configured. Use the skip flag if you want to disable execution. -> [Help 1
```

Reply:
Not sure why you are getting that error, it was correctly failing the build if I ran in the wrong version of maven.

Reply:
This is used by steps such as compile, for example ``mvn compile`` will fail if I set the maven version to one newer than installed on my computer:
```
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-enforcer-plugin:3.4.1:enforce (enforce-maven) on project geonetwork: 
[ERROR] Rule 0: org.apache.maven.enforcer.rules.version.RequireMavenVersion failed with message:
[ERROR] Detected Maven Version: 3.9.5 is not in the allowed range [4.0.9,).
```